packit/cli/build.py

```diff
@@ -86,27 +87,39 @@ def build(
         config=config, dist_git_path=dist_git_path, local_project=path_or_url
     )
 
+    if koji_target is None:
+        targets_to_build = {None}
+    else:
+        targets_to_build = get_koji_targets(koji_target)
+
     branches_to_build = get_branches(*dist_git_branch.split(","), default="master")
     click.echo(f"Building for the following branches: {', '.join(branches_to_build)}")
 
-    for branch in branches_to_build:
-        try:
-            out = api.build(
-                dist_git_branch=branch,
-                scratch=scratch,
-                nowait=nowait,
-                koji_target=koji_target,
-                from_upstream=from_upstream,
-            )
-        except PackitCommandFailedError as ex:
-            logs_stdout = "\n>>> ".join(ex.stdout_output.strip().split("\n"))
-            logs_stderr = "\n!!! ".join(ex.stderr_output.strip().split("\n"))
-            click.echo(
-                f"Build for branch '{branch}' failed. \n"
-                f">>> {logs_stdout}\n"
-                f"!!! {logs_stderr}\n",
-                err=True,
-            )
-        else:
-            if out:
-                print(ensure_str(out))
+    if len(targets_to_build) > 1 and len(branches_to_build) > 1:
+        raise PackitConfigException(
+            "Parameters --dist-git-branch and --koji-target cannot have"
```

Comment:
```suggestion
            "Parameters --dist-git-branch and --koji-target cannot have "
```

This is how it looks in terminal:
```
$ packit build --dist-git-branch fedora-all --koji-target fedora-stable
Building for the following branches: f32, master, f34, f33
ERROR    Parameters --dist-git-branch and --koji-target cannot havemultiple values at the same time.
```